tests/install_vm.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+import argparse
+import os
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument(
+    "--distro",
+    dest="distro",
+    default="fedora",
+    choices=("fedora", "centos7"),
+    help="What type of distribution to install"
+)
+parser.add_argument(
+    "--domain",
+    dest="domain",
+    required=True,
+    help="What name should the new domain have."
+)
+parser.add_argument(
+    "--disk-dir",
+    dest="disk_dir",
+    default="/var/lib/libvirt/images/",
+    help="Location of the VM qcow file."
+)
+parser.add_argument(
+    "--ram",
+    dest="ram",
+    default=2048,
+    type=int,
+    help="Amount of RAM configured for the VM."
+)
+parser.add_argument(
+    "--cpu",
+    dest="cpu",
+    default=2,
+    type=int,
+    help="Number of CPU cores configured for the VM."
+)
+parser.add_argument(
+    "--dry",
+    dest="dry",
+    action="store_true",
+    help="Print command line instead of triggering command."
+)
+
+parser = parser.parse_args()
```

Comment:
this makes me weep

Reply:
please do `args = parser.parse_args()`, avoid `parser.__dict__`, that's a nasty workaround. Make this legible please.

Reply:
Eh, yeah it's actually stupid right. Will fix. Unfortunately the `__dict__` is not so easily avoidable. It won't be accepted by the format function, afaict. But it might be more readable to add rest into `args` and call `__dict__` at the very end. What do you think?

Reply:
@mpreisler changed it - it's still not perfect, but I want to avoid need to be explicit when formatting the string. It wouldn't help readability.